npc/quests/quests_ein.txt

```diff
@@ -474,18 +474,482 @@ einbech,97,167,5	script	Cavitar	4_M_EINOLD,{
 
 //== Uwe Kleine :: ein_anoz ================================
 einbroch,215,180,6	script	Uwe Kleine#ein	4_M_03,{
+	function F_AboutEinbroch {
+		mes "[Uwe]";
+		mes "Einbroch was originally";
+		mes "built to support Einbech's";
+		mes "mining efforts. Because it's";
+		mes "small and crowded with people,";
+		mes "there's no room to build the ore refining factories over there.";
+		next;
+		mes "[Uwe]";
+		mes "Since Einbroch used to be";
+		mes "an empty lot, it was perfect";
+		mes "for building factories. That's";
+		mes "what my grandfather told me a";
+		mes "long time ago. Anyway, Einbroch quickly grew into a major city.";
+		next;
+		mes "[Uwe]";
+		mes "Now people think that this";
+		mes "city was never planned to be";
+		mes "just an extension of Einbech.";
+		mes "See that rampart over there?";
+		mes "It doesn't connect to Einbech at all! No protection for them...";
+		next;
+		mes "[Uwe]";
+		mes "It's like the government";
+		mes "lost all interest in Einbech.";
+		mes "Even the miners there have";
+		mes "been moving here to work in ";
+		mes "the factories. But more people hasn't made this city more lively.";
+		next;
+		mes "[Uwe]";
+		mes "Einbroch may look modern";
+		mes "and exciting now, but soon";
+		mes "you'll see that there's no sign";
+		mes "of warmth or life. So... Just don't live here in your old age.";
+		next;
+		
+		.@rand = rand(2); //Aegis Script doesn't use randomness, but is pretty random ...
```

Comment:
Well it doesn't, but it'd be longer if it weren't random.
There's just one case I'd need to treat specifically.
I'll show you a sketch of it in a bit.